glean-core/ios/Glean/Glean.swift

```diff
@@ -465,6 +470,19 @@ public class Glean {
         }
     }
 
+    /// Set the log_pings debug option,
+    /// when this option is `true` the pings that are successfully submitted get logged.
+    ///
+    /// - parameters:
+    ///     * value: The value of the option.
+    public func setLogPings(_ value: Bool) {
+        if self.isInitialized() {
+            glean_set_log_pings(value.toByte())
```

Comment:
Should we also update the cached `logPings` member variable here?  I think it should also be updated to reflect the state of ping logging in the core, in this case if we are initialized then it sets it in the core but the local variable doesn't get set.

Reply:
That would go against what we are doing for the `debugViewTag` and `setUploadEnabled`. I am not necessarily opposed to it, I just think if we do it for `logPings` we should do it for the others.

Reply:
I'm not sure we need to do this, since we should only hit the "uninitialized" part before... well, being initialized

Reply:
Works for me then!